Gemfile

```diff
@@ -2,21 +2,6 @@ source "https://rubygems.org"
 
 gemspec
 
-# for generating i18n files, gettext > 3.0 dropped ruby 1.8 support
-gem 'gettext', '>= 3.1.3', '< 4.0.0'
-
-group :test do
```

Comment:
We install RuboCop testing without development dependencies, but with test dependencies. So just moving everything to a development dependency broke the RuboCop.

Reply:
Ah ok that makes, sense, will fix it up. Thanks for looking again

Reply:
Ah so these all would need to be done as:
`gem.add_dependency` then since I was looking at the docs for `gemspec` and you can't do grouping like you could in the `Gemfile`

If that is correct, I will go ahead and get it changed up

Reply:
Yeah, in a gemspec you can only do `add_dependency` (which is runtime) and `add_development_dependency` which is devel and maps to the `development` bundler group from the gemfile

Reply:
fwiw, @ekohl is wrong, we do not skip `development` group when installing rubocop: https://github.com/theforeman/actions/blob/v0/.github/workflows/rubocop.yml